apps/rotm/behaviours/skip-step.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const isIncluded = (a, b) => a.some(v => b.includes(v));
+
+module.exports = (config) => {
+  return superclass => class extends superclass {
+    getValues(req, res, callback) {
+      super.getValues(req, res, (err, values) => {
+
+        if (config && config.steps) {
+          let completedSteps = values.steps;
+          const currentStep = req.form.options.route;
+          if (config.steps.includes(currentStep) === false) {
+            config.steps.unshift(currentStep);
+          }
```

Comment:
This allows someone to add the currentstep in the config or not.  It will deal with both scenarios